test/com/google/javascript/jscomp/PeepholeIntegrationTest.java

```diff
@@ -132,18 +109,12 @@ public void testFoldOneChildBlocksIntegration() {
   }
 
   public void testFoldOneChildBlocksStringCompare() {
-    // The expected parse tree has a BLOCK structure around the true branch.
-    assertResultString("if(x){if(y){var x;}}else{var z;}",
-        "if(x){if(y)var x}else var z");
+    fold("if(x){if(y){var x;}}else{var z;}", "if(x){if(y)var x}else var z");
```

Comment:
I assume you mean we're now doing AST comparison instead of comparing the literal bytes of the JS. In that case we can afford a little whitespace (`if (x) {` instead of `if(x){` etc.)